packages/next/build/webpack/plugins/react-loadable-plugin.ts

```diff
@@ -55,6 +55,11 @@ function buildManifest(
             file
           )
 
+          const css = chunk.files.filter(
```

Comment:
I wonder if it is guaranteed for chunk to have only 1 css file in it, that would simplify the code.

Reply:
it seem that mini-css-extract-plugin doesnt do "shared" styles, so ye, one file per chunk.

Will not remove that though.